spec/classes/mod/jk_spec.rb

```diff
@@ -36,26 +36,25 @@
         },
       }
     end
+
     it { is_expected.to compile }
     it { is_expected.to compile.with_all_deps }
+    expected_content = "# This file is generated automatically by Puppet - DO NOT EDIT\n"\
```

Comment:
@david22swan is this intended ?

Reply:
Yep, it failed the Rubocop merge test

Reply:
'it's' aren't allowed to be more than 5 lines longs.